integration-tests/tests/src/tests/sync/user.ts

```diff
@@ -317,7 +317,7 @@ describe.skipIf(environment.missingServer, "User", () => {
           .logIn(Realm.Credentials.emailPassword({ email: validEmail, password: validPassword }))
           .catch((err) => {
             expect(err.message).equals("invalid username/password");
-            expect(err.code).equals(50);
+            expect(err.code).equals(4349);
```

Comment:
Why did the error code change?  Was this a core upgrade?

Reply:
Yep.

Reply:
I guess that's a breaking change then :D

Reply:
Not as I see it. These are not documented, so technically not a part of our public API as per semantic versioning.

Reply:
Mmm...true.  Although if someone were to have code relying on this error code, it would need to be updated.

Reply:
In any case, this doesn't block this PR from being merged.  It was just a bit of a concern I had.